update: extend communications framework (#81)
thoth-tech/handbook: docs/communication/professional-communication.md

Communication extends beyond the words we speak. There are many different forms of communication, and it is important to build these skills. The types of communication identified at Thoth Tech are:

- oral communication
- written communication
- non-verbal communication
- digital communication

## Why is professional communication important?

Communication is at the centre of our work. All forms of communication can be used to inform, influence, educate, persuade, and respond. Our skill in communication can determined the outcome and next steps of a meeting or project.

- It can ensure you pick the right style for the audience.
- It is a desirable soft skill.
- It can help to avoid misunderstanding or disrespect.
- It can be inspiring.
- It can improve teamwork and working relationships.
- It can increase effectivity and productivity.

## What makes up professional communication?

There are four basic skills that are intertwined with each communication type. These skills make up effective communication:

- listening
- reading
- writing
- speaking

> Effective communication differs from professional communication. For communication to be professional it is important that it is:

Reviewer: ```suggestion
Effective communication differs from professional communication. For communication to be professional, it is important that it is:
```

- complete
- polite
- clear
- accurate
- precise
- considerate
- single-focused
- consistent
- bi-directional

## How can you improve your communication skills?

There are many barriers to professional communication. However, like any other skill, this can be learnt with time and practice. Some tips to improve your skills:

- Ask for feedback
- Learn from others
- Work on your active listening
- Work on your body language
- Acknowledge and respond to barriers
  - jargon
  - emotions
  - attention
  - perception
  - accessibility

## Professional communication at Thoth Tech

1. Use formal or semi-formal language when contacting someone for the first time and try to build rapport.

> 2. Be respectful of a person's time. We do not expect you to be avaliable all the time and neither should your peers.If you want someones attention schedule meetings in advance or give a heads up.

Reviewer: ```suggestion
2. Be respectful of a person's time. We do not expect you to be available all the time and neither should your peers. If you want someone's attention, schedule meetings in advance or give a heads up.
```

> 3. All company data should be shareable by default.

Reviewer: **suggestion:** We can use `1.` for all items in the order list. Markdown will render them in the correct order. This will avoid human error and make inserting new items less noisy (ie. we do not need to increment numbers)

4. Use emoticons to brighten the mood but be sparing.
5. Provide meaningful responses in an appropriate time frame.
6. Provide clear, direct, and accurate information to avoid miscommunication.
7. Ask questions. Do them in a way that allows many people to see or respond.
8. Use Pull Requests or public chat channels instead of direct messages.
9. Handle conflicts with diplomacy and an open-mind.
10. Avoid controversial topics and gossip.

> 11. Offer [praise](/docs/communication/fearless-feedback.md#shoutouts) and [fearless feedback](/docs/communication/fearless-feedback.md#fearless-feedback).

Reviewer: ```suggestion
11. Offer [praise](fearless-feedback.md#shoutouts) and [fearless feedback](fearless-feedback.md#fearless-feedback).
```
